### PR TITLE
Angular interface

### PR DESCRIPTION
**Pull request by warphole**
Good day! Implemented user interface, authentication, guards, not found pages!
Most of the commits are in the branch [angular_start](https://github.com/asadbek2021/piggy-bank/tree/angular_start), because I have forgot to switching into a new branch and implemented all changes there!

### CODE REVIEW

File: client/todo-list/src/app/app-routing.module.ts [hunk 1]
```diff
@@ -1,10 +1,16 @@
 import { NgModule } from '@angular/core';
 import { RouterModule, Routes } from '@angular/router';
+import { NotFoundComponent } from './not-found/not-found.component';
 
-const routes: Routes = [];
+
+
+const routes: Routes = [
+  {path:'', redirectTo:'/todo', pathMatch:'full'},
+  {path:'**', component:NotFoundComponent},
```
Comment (devbarden) [inline client/todo-list/src/app/app-routing.module.ts:9]: bad code style. add spaces please

File: client/todo-list/src/app/app.component.ts [hunk 2]
```diff
@@ -7,4 +10,15 @@ import { Component } from '@angular/core';
 })
 export class AppComponent {
   title = 'todo-list';
+  constructor(private authService:AuthService, private router:Router) { }
+
+  get isLoggedIn():boolean {
+    return this.authService.isLoggedIn();
+  }
```
Comment (devbarden) [inline client/todo-list/src/app/app.component.ts:17]: please add empty line between methodos

**Review by devbarden — APPROVED** [review]
Looks good just bad code style regarding spaces and tabs. Please fix it

File: client/todo-list/src/app/app-routing.module.ts (hunk 1 above)
Comment (asadbek2021) [reply client/todo-list/src/app/app-routing.module.ts:9]: ok, thanks

**Pull request merged**